src/app/lib/README.md

```diff
@@ -100,6 +100,26 @@ Therefore to meet all requirements we implement the following CSS grid where gut
 - `6 + 2 of 0px` - is based on having a 0px column either side of UX grid so we can use CSS grid-gaps as the UX margin while allowing a grid item to be able to go "full bleed".
 - `6 + 2 of 8px` - between 400px and 599px we have an 8px column either side of the UX grid so that the grid-gap (8px) and additional column (8px) combined meet the 16px UX margin.
 
+### Nested grid usage
+
+As of [#1680](https://github.com/bbc/simorgh/pull/1680) `GridItem`s now also have their own internal grid which maps to the external grid they occupy.
+
+<img width="1280" alt="A screenshot of nested grid items" src="https://user-images.githubusercontent.com/19362408/57520118-cdc09000-7314-11e9-8657-d5d798c0b018.png"> 
```

Comment:
👍 Thank you for writing this up! Great explanation. I particularly like the usage examples in the table.

Reply:
Are we sure we want every component to have an internal grid at all times? I assumed we'd have a `const` that was exported containing `styled.css` which could be added to the components that need to know about the internal grid. Feels like a lot of CSS for the browser to process if for example every paragraph block has an inner grid?

Reply:
documentation needs updating so I'm moving this back to PR in progress.